media/starboard/starboard_renderer.h

```diff
@@ -59,11 +59,37 @@ class MEDIA_EXPORT StarboardRenderer final : public Renderer,
     // TODO(b/380935131): Consider to implement `LatencyHint` for SbPlayer.
     NOTIMPLEMENTED();
   }
+  void SetPreservesPitch(bool preserves_pitch) final {
+    if (!preserves_pitch) {
+      LOG(WARNING)
+          << "Calling SetPreservesPitch() with preserves_pitch=false is "
+             "not supported.";
+      return;
+    }
+  }
+  void SetWasPlayedWithUserActivation(
+      bool was_played_with_user_activation) final {
+    if (was_played_with_user_activation) {
+      LOG(WARNING) << "Calling SetWasPlayedWithUserActivation() with "
+                      "was_played_with_user_activation=true is not supported.";
+      return;
+    }
+  }
   void Flush(base::OnceClosure flush_cb) final;
   void StartPlayingFrom(base::TimeDelta time) final;
   void SetPlaybackRate(double playback_rate) final;
   void SetVolume(float volume) final;
   base::TimeDelta GetMediaTime() final;
+  void OnSelectedVideoTracksChanged(
+      const std::vector<DemuxerStream*>& enabled_tracks,
+      base::OnceClosure change_completed_cb) final {
+    LOG(WARNING) << "Calling OnSelectedVideoTracksChanged() is not supported.";
```

Comment:
We need to call `change_completed_cb`, and this shouldn't be a warning.  Consider using LOG(INFO) and then delegate to Renderer::OnSelectedVideoTracksChanged), where `change_completed_cb` will be called.
The same to the function below.

Reply:
Done.